.github/workflows/release.yml

```diff
@@ -5,45 +5,83 @@ on:
     types: [created]
 
 jobs:
-  release:
+  build:
     if: startsWith(github.ref, 'refs/tags/v')
     runs-on: ubuntu-latest
     steps:
-      - name: set env
-        run: echo "RELEASE_TAG=${GITHUB_REF#refs/tags/}" >> $GITHUB_ENV
       - uses: actions/checkout@v3
         with:
           submodules: recursive
-      # Setup .npmrc file to publish to npm
       - uses: actions/setup-node@v3
         with:
           node-version: '16.x'
           registry-url: 'https://registry.npmjs.org'
       - run: npm install
       - name: grunt release
         run: ./node_modules/.bin/grunt release
+      - uses: actions/upload-artifact@v3
+        with:
+          name: dist
+          path: dist/
+      - uses: actions/upload-artifact@v3
+        with:
+          name: docs
+          path: docs/
+  publish-to-github:
```

Comment:
Ah nice we don't have to manually upload artifacts to GitHub release, I should apply it to ion-hive-serde release too. I didn't use gh-cli in GHA before, do we need authentication somewhere first or the corresponding workflow will recognize it?

Reply:
You authenticate by putting `GITHUB_TOKEN` in the `env` of the `run` step where you are using the `gh` CLI.

Reply:
Thanks, found it.